packages/@aws-cdk/aws-appmesh/lib/virtual-service.ts

```diff
@@ -77,9 +63,9 @@ export interface VirtualServiceBaseProps {
  */
 export interface VirtualServiceProps extends VirtualServiceBaseProps {
   /**
-   * The Mesh which the VirtualService belongs to
+   * The VirtualNode or VirtualRouter which the VirtualService uses as its provider
    */
-  readonly mesh: IMesh;
+  readonly virtualServiceProvider: VirtualServiceProvider;
```

Comment:
Now that you've removed `addVirtualService()` from `IMesh`, I'm pretty sure the `VirtualServiceBaseProps` is not used anywhere. If I'm right, get rid of the it in favor of `VirtualServiceProps`.